evm-adapters/src/sputnik/cheatcodes/mod.rs

```diff
@@ -80,3 +80,191 @@ ethers::contract::abigen!(
 );
 
 ethers::contract::abigen!(Console, "./testdata/console.json",);
+
+// Manually implement log(uint) and log(int),
```

Comment:
Will this work for other signatures that use `uint` besides these two like `log(uint,string)`?

Reply:
uhh, yikes, haven't thought about that
I'm afraid this won't work because those cases all suffer from the same uint256/unit problem...

I'll need to do some tests inside abigen first